nncf/openvino/graph/metatypes/groups.py

```diff
@@ -46,34 +47,38 @@
 ]
 
 
-QUANTIZABLE_OPERATIONS = [
+INPUTS_QUANTIZABLE_OPERATIONS = [
```

Comment:
@KodiaqQ @alexsu52 I noticed that `ov_metatypes.OVDeformableConvolutionMetatype` is missing here, but it is present in `LINEAR_OPERATIONS` list and consequently used in fused patterns. So, my question is whether this is expected behavior and we should not quantize the `DeformableConvolution` operation, or if it is a bug that should be fixed?"

Reply:
Never met `DeformableConvolution` in the OpenVINO models. Need to an example.

Reply:
It isn't a bug. `DeformableConvolution` should be fused but runtime doesn't support INT8 execution (ticket: 106633)